ngraph/frontend/paddlepaddle/src/op_table.cpp

```diff
@@ -8,10 +8,27 @@
 #include "op/clip.hpp"
 #include "op/concat.hpp"
 #include "op/conv2d.hpp"
+#include "op/conv2d_transpose.hpp"
+#include "op/dropout.hpp"
 #include "op/elementwise_ops.hpp"
+#include "op/equal.hpp"
+#include "op/expand_v2.hpp"
+#include "op/fill_constant.hpp"
+#include "op/fill_constant_batch_size_like.hpp"
+#include "op/flatten_contiguous_range.hpp"
+#include "op/greater_equal.hpp"
+#include "op/hard_sigmoid.hpp"
+#include "op/hard_swish.hpp"
+#include "op/interp.hpp"
+#include "op/leakyrelu.hpp"
+#include "op/log.hpp"
+#include "op/logical_not.hpp"
+#include "op/matmul.hpp"
```

Comment:
Can we use a forward declaration for the functions instead of header file with a single function?

Reply:
@nosovmik What do you think ?

Reply:
@nosovmik ping.

We have a lot of dummy .hpp files with declarations only..

Reply:
@ilya-lavrenov Do you suggest to move all of them to single file, like op_table.hpp?
It would be easier to read, however each operation, like log.cpp, will need to include "op_table.hpp" which will have a lot of declarations not needed for log.cpp.
I'm basically ok with both approaches

Reply:
I suggest in current file:
```c++
namespace ngraph
{
    namespace frontend
    {
        namespace pdpd
        {
            namespace op
            {
                NamedOutputs greater_equal(const NodeContext& node);
                NamedOutputs transpose2(const NodeContext& node);
                // ..
            }
        } // namespace pdpd
    }     // namespace frontend
} // namespace ngraph 
```
So, we don't need any hpp files with single declaration

Reply:
Or we can use an approach like https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/cldnn_engine/ops/scatter_elements_update.cpp#L66

here we don't need a lot of .hpp files as well and each operation registers itself

Reply:
Yes, I guess something like dynamic ops registration may be needed in future if we'll allow to dynamically register some custom user-defined operations, but I guess not now :-)

Reply:
We don't talk about custom operations here. There is a way how we can register our operations handlers without creation of multiple hpp files. No more

Reply:
@zhangYiIntel We have short discussion, can you try the following approach (https://github.com/openvinotoolkit/openvino/pull/6632#discussion_r676377975)
1) Remove all op/*.hpp files
2) Add declarations only in op_table.cpp file as these are not needed anywhere else
3) In every op/*.cpp file - just include `#include <node_context.hpp>" as operation definition doesn't need to know it's forward declaration

Reply:
@nosovmik No Problem, I will have a try.

Reply:
Fix in 5caa477f935e11eaab789c301af16866d6ad89b7